### ISSUE TITLE
Shops List sync, spamming console

### ISSUE THREAD

**Issue by Paul-DK** [myzticbean/QSFindItemAddOn#29]
Since the latest update, `Shops List sync complete` has been spamming my console every 10 - 20 mins. I'd love for it to not do this, please :)

```
[13:08:20 INFO]: [QSFindItemAddOn] [ASYNC] Shops List sync complete. Time took: 0ms.
[13:23:20 INFO]: [QSFindItemAddOn] [ASYNC] Shops List sync complete. Time took: 0ms.
[13:38:20 INFO]: [QSFindItemAddOn] [ASYNC] Shops List sync complete. Time took: 0ms.
[13:53:20 INFO]: [QSFindItemAddOn] [ASYNC] Shops List sync complete. Time took: 0ms.
[14:08:20 INFO]: [QSFindItemAddOn] [ASYNC] Shops List sync complete. Time took: 0ms.
[14:23:20 INFO]: [QSFindItemAddOn] [ASYNC] Shops List sync complete. Time took: 0ms.
[14:38:20 INFO]: [QSFindItemAddOn] [ASYNC] Shops List sync complete. Time took: 0ms.
[14:53:20 INFO]: [QSFindItemAddOn] [ASYNC] Shops List sync complete. Time took: 0ms.
[15:08:20 INFO]: [QSFindItemAddOn] [ASYNC] Shops List sync complete. Time took: 0ms.
[15:23:20 INFO]: [Vault] Checking for Updates ... 
[15:23:20 INFO]: [Vault] No new version available
[15:23:20 INFO]: [QSFindItemAddOn] [ASYNC] Shops List sync complete. Time took: 1ms.
[15:38:20 INFO]: [QSFindItemAddOn] [ASYNC] Shops List sync complete. Time took: 0ms.
[15:53:20 INFO]: [QSFindItemAddOn] [ASYNC] Shops List sync complete. Time took: 0ms.
[16:08:20 INFO]: [QSFindItemAddOn] [ASYNC] Shops List sync complete. Time took: 0ms.
```

**Comment by myzticbean** [comment]
These log statements were present before this build as well with different wordings, but I will move them to debug-mode only in the next release build so it wont fill up your console.

**Comment by Paul-DK** [comment]
Thank you!!

**Comment by myzticbean** [comment]
Fixed in v2.0.6.1
